bootstrapvz/plugins/puppet/tasks.py

```diff
@@ -36,28 +55,82 @@ def run(cls, info):
             raise TaskError(msg)
 
 
-class AddPackages(Task):
-    description = 'Add puppet package'
-    phase = phases.preparation
+class InstallPuppetlabsPC1RepoPackage(Task):
```

Comment:
Well, i tried to get it *working* and i had some trouble (or fatigue)  implementing this step with the InstallTrustedKeys task. It threw me some crazy gpg errors.. Will create an issue to address this with InstallTrustedKeys. this however also works...